cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.adoc

```diff
@@ -1,28 +1,45 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="init-bundle-cloud-ocp-generate"]
-= Generating an init bundle for secured clusters
+= Generating an init bundle or cluster registration secret for secured clusters
 include::modules/common-attributes.adoc[]
 :context: init-bundle-cloud-ocp-generate
 
 toc::[]
 
 [role="_abstract"]
 
-Before you install the `SecuredCluster` resource on a cluster, you must create an init bundle. The cluster that has `SecuredCluster` installed and configured then uses this bundle to authenticate with Central. You can create an init bundle by using either the {product-title-short} portal or the `roxctl` CLI. You then apply the init bundle by using it to create resources.
+Before you set up a secured cluster, you must create an init bundle or cluster registration secret (CRS). The secured cluster then uses this bundle or CRS to authenticate with the Central instance, also called Central. You can create an init bundle or CRS by using either the {product-title-short} portal or the `roxctl` CLI. You then apply the init bundle or CRS by using it to create resources.
 
 [NOTE]
 ====
 You must have the `Admin` user role to create an init bundle.
 ====
 
+{product-title-short} uses a special artifact during installation that allows the {product-title-short} Central component to communicate securely with secured clusters that you are adding. Before release 4.7, {product-title-short} used _init bundles_ exclusively for initiating the secure communication channel. Beginning with 4.7, {product-title-short} provides an alternative to init bundles called _cluster registration secrets_ (CRSes).
+
+:FeatureName: Cluster registration secrets
+include::snippets/technology-preview.adoc[]
+
+include::snippets/cluster-registration-secrets.adoc[]
+
+Currently, you can use either an init bundle or a cluster registration secret (CRS) during installation of a secured cluster. However, {product-title-short} does not yet provide a way to create a CRS by using the portal. Therefore, you must create the CRS by using the `roxctl` CLI.
```

Comment:
"Currently" suggests things might change, which I am always wary of documenting. 
```suggestion
You can use either an init bundle or a cluster registration secret (CRS) during installation of a secured cluster. However, {product-title-short} does not yet provide a way to create a CRS by using the portal. Therefore, you must create the CRS by using the `roxctl` CLI.
```